radio/src/targets/taranis/hal.h

```diff
@@ -1313,6 +1313,7 @@
   #define ADC_GPIO_PIN_SLIDER2          LL_GPIO_PIN_5  // PC.05
   #define ADC_GPIO_PIN_BATT             LL_GPIO_PIN_0  // PC.00
   #define ADC_GPIOA_PINS                (ADC_GPIO_PIN_STICK_RV | ADC_GPIO_PIN_STICK_RH | ADC_GPIO_PIN_STICK_LV | ADC_GPIO_PIN_STICK_LH | ADC_GPIO_PIN_POT1)
+  #define ADC_GPIOA_PINS_FS             (ADC_GPIO_PIN_STICK_LH | ADC_GPIO_PIN_STICK_LV | ADC_GPIO_PIN_POT1)
```

Comment:
I don't think this is needed anymore.

Reply:
Yes, I think so.  Well, why it is still there. I remembered that I removed these.changes.during rebase.....